geomstats/hyperbolic_space.py

```diff
@@ -115,7 +109,7 @@ def projection_to_tangent_space(self, vector, base_point):
 
     def intrinsic_to_extrinsic_coords(self, point_intrinsic):
         """
-        From the intrinsic coordinates in the hyperbolic space,
+        Convert from intrinsic coordinates in the hyperbolic space,
```

Comment:
was actually bette rbefore. If you use convert say what you actually convert. Point, vector, potatoes?